flopy/mf6/modflow/mfsimulation.py

```diff
@@ -1009,32 +1019,44 @@ def load_package(
             parent_package._add_package(package, package.path)
         return package
 
-    def register_ims_package(self, ims_file, model_list):
+    def register_ims_package(self, solution_file, model_list):
+        warnings.warn(
+            "register_ims_package() has been deprecated and will be "
+            "removed in version 3.3.7. Use "
+            "register_solution_package() instead.",
+            DeprecationWarning,
+        )
+        self.register_solution_package(solution_file, model_list)
+
```

Comment:
I don't think we want to deprecate this yet.  Can we keep it for now, and reconsider when we officially release another solver type?

Reply:
@langevin-usgs, why not give the user as much warning as possible when a method will be deprecated?  Are you thinking that you might come up with a better name than "register_solution_package" and don't want to commit to that name yet?

Reply:
EMS is experimental now and while we're pretty sure it will come out at some point, it's premature to talk about deprecating existing options in order to support it.  For the medium term, I don't see us having more than two types of solutions, so continuing to support register_ims_package() seems fine for now.

Reply:
@langevin-usgs, deprecation warning removed